File: library/src/main/java/me/shkschneider/skeleton/java/ListHelper.java
package me.shkschneider.skeleton.java;

import android.support.annotation.NonNull;
import android.support.annotation.Size;

import java.util.List;

public class ListHelper {

    protected ListHelper() {
        // Empty
    }

    public static <T> boolean contains(@NonNull final List<T> objects, @NonNull final T object) {
        return objects.contains(object);
    }

    public static <T> T first(@Size(min=1) @NonNull final List<T> objects) {
        return objects.get(0);
    }

    public static <T> T last(@Size(min=1) @NonNull final List<T> objects) {
        return objects.get(objects.size() - 1);
    }

}

File: library/src/main/kotlin/me/shkschneider/skeleton/java/ListHelper.kt
package me.shkschneider.skeleton.java

import android.support.annotation.Size

object ListHelper {

    fun <T> contains(objects: List<T>, obj: T): Boolean {
        return objects.contains(obj)
    }

    fun <T> first(@Size(min = 1) objects: List<T>): T {
        return objects[0]
    }

    fun <T> last(@Size(min = 1) objects: List<T>): T {
        return objects[objects.size - 1]
    }

}
